src/modules/vtol_att_control/standard.cpp

```diff
@@ -385,13 +397,15 @@ void Standard::fill_actuator_outputs()
 		mc_out[actuator_controls_s::INDEX_THROTTLE]     = mc_in[actuator_controls_s::INDEX_THROTTLE] * _mc_throttle_weight;
 		mc_out[actuator_controls_s::INDEX_LANDING_GEAR] = 0;
 
-		// FW out = FW in, with VTOL transition controlling throttle and airbrakes
+		// FW out = FW in, with VTOL transition controlling throttle, flaperons airbrakes (for reverse thrust).
 		fw_out[actuator_controls_s::INDEX_ROLL]         = fw_in[actuator_controls_s::INDEX_ROLL];
 		fw_out[actuator_controls_s::INDEX_PITCH]        = fw_in[actuator_controls_s::INDEX_PITCH];
 		fw_out[actuator_controls_s::INDEX_YAW]          = fw_in[actuator_controls_s::INDEX_YAW];
 		fw_out[actuator_controls_s::INDEX_THROTTLE]     = _pusher_throttle;
 		fw_out[actuator_controls_s::INDEX_FLAPS]        = fw_in[actuator_controls_s::INDEX_FLAPS];
-		fw_out[actuator_controls_s::INDEX_AIRBRAKES]    = _reverse_output;
+		fw_out[actuator_controls_s::INDEX_SPOILERS]     = _reverse_output;
```

Comment:
We would need to change https://github.com/PX4/PX4-Autopilot/blob/b7e0f17c6a845d1766ee21e7d5e26af4f998442e/ROMFS/px4fmu_common/mixers/deltaquad.main.mix#L47. The delta quad seems to be the only vehicle with active braking during backtransition. @sanderux that's still enabled like that on the DQ?

It's not nice that the index changes if people are using it like this currently. And wouldn't it actually be more correct to add the functionality you add here to INDEX_SPOILERS? As raising the ailerons to reduce lift/increase drag is closer to what spoilers usually do no?

Reply:
I wasn't a fan of changing the index and went back and forth on this. I ultimately did it only because that is what it is on fixed wing and to keep it consistent across the two. At the end of the day it doesnt really matter to me I just finally thought that if we are going to do it one way on one vehicle type, best to keep it the same but yes - spoilers are definitely closer than airbrakes. I would want to just check that any sort of coupling for landing as a fixed wing still is in tack if decided to land as one (since that is an option) [https://github.com/PX4/PX4-Autopilot/commit/7fc97ed147222668d996815e0feab887f9abf7ed] this refers to everything as flaperons and airbrakes so likely we should get these consistent as well once this side is settled.

Cool - re the descent aspect. Do you envision it during a straight decent or something that is banked? You can definitely slow down quicker just trying to understand - the problem with the FW_LND_EARLYCFG parameter was that if flaps and ailerons were deployed very early and you end up circling for a while to reach a target altitude as well as have the flaps and ailerons set to large amounts of travel then it is possible to actually stall the aircraft if you were making too much drag. Full flaps were fine but if set too high, full flaps and almost full ailerons up got tricky when you were banking (increase g load and increase min stall speed). Yes, you were making more lift so that would also increase the min needed speed but I saw in some tests that it would getting close to being a bit on the edge so it really needed to be enabled carefully. Decreased speed right around stall speed saw decrease authority. 

If you enable it while going straight and down, it was just like throwing a parachute out the back to slow down but all was ok. If I was going to revisit that I would add another parameter that if speed hit below or above X (percentage of min speed), then flaps and ailerons (or just ailerons) retracted until speed increased since it was really the flaps and ailerons at the same time that caused the increase in drag.

All that said, it is the easiest way to slow down without adding functionality to the aircraft and utilizing what is already there.

Reply:
I think we should fix the naming as we're on it.

See https://github.com/PX4/PX4-Autopilot/pull/19329 for a proposal. Next up would be to clean up the term "flaperon" in the fixed-wing controller. 

>Do you envision it during a straight decent or something that is banked? 

Also while banked in the loiter down. But a full deflection will prob not work, so I guess we need to add yet another scaling/setpoint parameter for it. Thanks for sharing your learning, I will come back to them when it's about time to tune it myself!